src/main/java/de/tum/in/test/api/ast/asserting/UnwantedNodesAssert.java

```diff
@@ -34,8 +34,18 @@ public class UnwantedNodesAssert extends AbstractAssert<UnwantedNodesAssert, Pat
 	 */
 	private final LanguageLevel level;
 
+	/**
+	 * Whether to enable the main method for the Java parser Default is set to false
```

Comment:
Can you be a bit more precise here? I'm note sure what "enable the main method for the Java parser" is supposed to mean. Does this mean that this method should not be checked? What about submissions with multiple main methods?

Reply:
In addition to that, I am not a fan of unsafe defaults. Again, more rationale needed for use case, requirements, potential conflicts and alternative solutions.

Reply:
I now saw that main is included by default in the code. The comment here just does not match.

Reply:
I will be improving the comments overall thanks for the nice catch!